docs/pages/insights/insights-header.tsx

```diff
@@ -281,7 +301,32 @@ const PeriodSelect: FC<PeriodSelectProps> = memo(() => {
         gap="sm"
         pt="sm"
       >
-        {INSIGHT_PERIOD_SUGGEST.map((value) => {
+        {INSIGHT_MONTH_PERIOD_SUGGEST.map((value) => {
+          return (
+            <Center
+              key={value}
+              as="button"
+              type="button"
+              bg={["blackAlpha.100", "whiteAlpha.100"]}
+              fontSize="sm"
+              h="8"
+              rounded="md"
+              transitionDuration="slower"
+              transitionProperty="background"
+              w="full"
+              _hover={{
+                bg: ["blackAlpha.200", "whiteAlpha.200"],
+              }}
+              onClick={() => onSuggestChange(value)}
+            >
+              {t(`insights.period.${value}`)}
+            </Center>
+          )
+        })}
+
+        <Divider />
```

Comment:
The color of the border is different from the others.

Reply:
@hirotomoyamada 
Which border color?

Reply:
It's `["blackAlpha.200", "whiteAlpha.100"]`.

Reply:
@hirotomoyamada 
Fixed.